services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/utils/AttributeValueHelper.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.dynamodb.utils;
+
+import java.nio.ByteBuffer;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+public final class AttributeValueHelper {
```

Comment:
Can we comment this class and all of its methods? Not sure what it does.

What's the difference between a `Helper` and `Utils`? 

Could we have a way to generate these onto the model? That would make them a lot more discoverable, and we were talking about doing that for our high-level libraries going forward. (`AttributeValue.create(Number)`). This is a high-level library.